mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -104,8 +107,8 @@ private[classification] trait LogisticRegressionParams extends ProbabilisticClas
    * @group setParam
    */
   def setThresholds(value: Array[Double]): this.type = {
-    if (isSet(threshold)) clear(threshold)
-    set(thresholds, value)
+    logWarning("Ignoring setThresholds(), use setThreshold() for binary Logistic Regression.")
```

Comment:
+1 @sethah . I think it will have different meanings compared with the existing ```setThresholds``` when it will be re-enabled for supporting multi-class logistic regression. So it's better to remove the note entirely or just left the doc "It will be re-enabled once multi-class logistic regression is included".